qln/msghandler.go

```diff
@@ -53,6 +47,28 @@ func (nd *LitNode) PeerHandler(msg lnutil.LitMsg, q *Qchan, peer *RemotePeer) er
 
 }
 
+// BroadcastMessage sends a chat message to all connected websocket clients
+func (nd *LitNode) BroadcastChatMessage(msg lnutil.LitMsg) error {
+	chat, ok := msg.(lnutil.ChatMsg)
+	if !ok {
+		return fmt.Errorf("can't cast to chat message")
+	}
+
+	// get the current set of channels and save them, they may be deleted by litrpc
+	// before the iteration is over
+	var channels []chan lnutil.ChatMsg
```

Comment:
it shouldn't, the concern is with the `range nd.UserWsCons` where nd.UserWsCons changes during the iteration
`range channels` doesn't have the same problem